requirements.txt

```diff
@@ -1,5 +1,8 @@
 gym
 numpy>=1.15
-pytest
-tensorflow  # Replace with tensorflow-gpu if you have a GPU
+tensorflow>=1.13.1,<2  # Replace with tensorflow-gpu if you have a GPU
 tqdm
+git+git://github.com/hill-a/stable-baselines.git@e59f32#egg=stable-baselines
```

Comment:
I'm curious how you got this to work.

Reply:
I had to make this patch to Stable Baselines: https://github.com/hill-a/stable-baselines/pull/240